db/migrate/20210512025422_remove_settings_mascots_table.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSettingsMascotsTable < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :settings_mascots
+  end
+end
```

Comment:
I know we're not going back but I wonder if we should use the block syntax so that the migration is reversible:

```ruby
drop_table :settings_mascots do |t|
  t.text "value"
  # ...
end
```

Reply:
I'm pretty sure we already have other non-reversible migrations so I'd rather save myself the effort. But I can add it if you want.

Reply:
nope, not really blocking :D

Reply:
TIL `drop_table` accepts a block. Kinda weird, though, considering it's not a truly reversible migration.